tasks/src/main/scala/dagr/tasks/vc/VarDictJava.scala

```diff
@@ -50,13 +51,13 @@ object VarDictJava extends Configuration {
   val RootDirConfigKey = "vardictjava.dir"
 
   /** Path to the VarDictJava directory. */
-  lazy val RootDir = configure[Path](RootDirConfigKey)
+  val RootDir = configure[Path](RootDirConfigKey)
```

Comment:
I made these non-lazy so that any configuration issues will show up at the first reference to the class/object instead of much later when the tasks try to run.  I find it much more helpful for debugging new setups.